data/json/faults.json

```diff
@@ -0,0 +1,78 @@
+[
+  {
+    "id": "fault_engine_belt_drive",
+    "type": "fault",
+    "name": "drive belt",
+    "description": "Required for operation of an attached alternator.",
+    "skills": [ [ "mechanics", 2 ] ],
+    "requirements": {
+      "qualities": [ { "id": "WRENCH", "level": 1 } ],
+      "components": [ [ [ "drivebelt", 1 ], [ "drivebelt_makeshift", 1 ] ] ]
+    }
+  },
+  {
+    "id": "fault_engine_glow_plug",
+    "type": "fault",
+    "name": "glow plugs",
+    "description": "Required to start the vehicle in low ambient tmperatures.",
```

Comment:
It's not required but very helpful in cold weather. See the code comments at `vehicle.cpp:1487`
```
// non-linear range [100-1000]; f(0.0) = 100, f(0.6) = 250, f(0.8) = 500, f(0.9) = 1000
// diesel engines with working glow plugs always start with f = 0.6 (or better)
```
Effectively having glow plugs limits `engine_cold_factor` to a maximum of `0.6`. In summer there's probably not much in it, in winter it's still possible to start the engine but you'll it'll take a few minutes and flatten the battery somewhat.